### DISCUSSION TITLE
keeps on crashing and windows atuo repair loop

### DISCUSSION THREAD

**Discussion by kle3q** [utmapp/UTM#2589] [UTM Help (macOS)]
I want windows on my Mac so I searched that up and so umt so I download every thing and made the virtual machine and the virtual machine works it crashes please fix this

**Comment by conath** [comment]
Hi kle3q. GitHub issues is for discussing technical issues related to UTM application, not your Windows virtual machine. So I'm moving this to GitHub Discussions. 

Regarding your "Windows auto repair loop", this reminds me of a problem that has been reported many times before. Are you using the VMDK from Microsoft? Unfortunately, this file is [not compatible with UTM's technology (QEMU)](https://github.com/utmapp/UTM/issues/185) and this means you will see the corrupted Windows. The command to convert the VMDK to the preferred QOW2 format is:

`qemu-img convert -p -O qcow2 /path/to/Windows10_InsiderPreview_Client_ARM64_en-us_21354.VHDX /path/to/WindowsARMImage.qcow2`

You can install `qemu-img` by using `brew install qemu`. [More info here](https://thatsniceguy.gitbook.io/utmfaq/software-and-vms/misc-tips#vhdx-to-qcow2-conversion).

Note that if the Windows is already corrupted, the conversion will not help. You need to download the VMDK again and the convert it before you load it in UTM.
